test/test.js

```diff
@@ -383,6 +393,26 @@ describe('Layer', () => {
       geotiff);
   });
 
+  it('can get layers by workspace', async () => {
+    const result = await grc.layers.getLayers(workSpace);
+    expect(result.layers.layer.length).to.equal(3);
+  });
+
+  it('works with non-existing workspaces', async () => {
+    const result = await grc.layers.getLayers('fantasy-workspace');
+    expect(result.layers).to.equal('');
```

Comment:
Is this correct by having an empty string (`''`) as expectation value?

Reply:
true. fixed in 30c30d4